dockstore-webservice/src/test/java/io/dockstore/webservice/helpers/GitHubHelperTest.java

```diff
@@ -12,6 +13,20 @@
 
 public class GitHubHelperTest {
 
+    private static final String CODE = "abcdefghijklmnop";
+    private static final String GITHUB_CLIENT_ID = "123456789abcd";
+    private static final String GITHUB_CLIENT_SECRET = "98654321dcba";
+
+    @Test
+    public void testGetGitHubAccessToken() {
+        try {
+            GitHubHelper.getGitHubAccessToken(CODE, GITHUB_CLIENT_ID, GITHUB_CLIENT_SECRET);
```

Comment:
You should add a `fail()` after this line; if the function didn't throw an exception, your test will pass, which isn't what you want.